temples.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Temples Assignment</title>
+    <meta name="description" content="This is a website for a excersise of a interactive album that works in mobile webpage and normal notebook/window page for student Andre Linhares to repository WDD131">
+    <meta name="author" content="Andre Linhares de Souza">
+    <link rel="stylesheet" href="styles/temples.css">
+    <link rel="stylesheet" href="styles/temples-large.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <script src="scripts/temples.js" defer></script>
+</head>
+
+<body>
+    <header class="header">
+        <div>
+            <h1 class="title-temple">Temple Album</h1>
+            <button class="menu-button" onclick="toggleMenu()">☰</button>        
+        </div>
+    
+            <nav class="menu" id="menu" style="display: flex;">
```

Comment:
Remove this inline style. It is causing your navigation to be visible when it is supposed to be hidden.